### ISSUE TITLE
Wrong indentation in antiquotation

### ISSUE THREAD

**Issue by basvandijk** [nix-community/nixpkgs-fmt#175]
## Input
```nix
{
  foo = ''
    bar = ${builtins.concatStringsSep " " [
      1
      2
      3
    ]}
    bla = hoi
  '';
}
```

## Output
```nix
{
  foo = ''
    bar = ${builtins.concatStringsSep " " [
    1
    2
    3
  ]}
    bla = hoi
  '';
}

```

## Desired output

Same as the input:
```nix
{
  foo = ''
    bar = ${builtins.concatStringsSep " " [
      1
      2
      3
    ]}
    bla = hoi
  '';
}
```

**Comment by basvandijk** [comment]
It seems that the Nix code in the antiquotation is indented to the indentation of the outer Nix code. What should probably happen is that the indentation in antiquotations is reset to the indentation of the first non whitespace character in the multi-line string.

**Comment by zimbatm** [comment]
Yeah there is something wrong with the indentation engine.

**Comment by domenkozar** [comment]
I saw the same with

```nix
            ${ optionalString cfg.isHTTPS ''
          ssl_certificate ${cfg.fullChain};
          ssl_certificate_key ${cfg.privateKey};
        '' }
```

**Comment by Rizary** [comment]
Fixed by #187
